src/main/sensors/gyro.c

```diff
@@ -588,10 +617,19 @@ void gyroUpdateSensor(gyroSensor_t *gyroSensor)
     gyroSensor->gyroDev.dataReady = false;
 
     if (isGyroSensorCalibrationComplete(gyroSensor)) {
-        // move gyro data into 32-bit variables to avoid overflows in calculations
-        gyroSensor->gyroDev.gyroADC[X] = (int32_t)gyroSensor->gyroDev.gyroADCRaw[X] - (int32_t)gyroSensor->gyroDev.gyroZero[X];
-        gyroSensor->gyroDev.gyroADC[Y] = (int32_t)gyroSensor->gyroDev.gyroADCRaw[Y] - (int32_t)gyroSensor->gyroDev.gyroZero[Y];
-        gyroSensor->gyroDev.gyroADC[Z] = (int32_t)gyroSensor->gyroDev.gyroADCRaw[Z] - (int32_t)gyroSensor->gyroDev.gyroZero[Z];
+    	// move 16-bit gyro data into 32-bit variables to avoid overflows in calculations
+
+    	gyroSensor->gyroDev.gyroADC[X] = gyroSensor->gyroDev.gyroADCRaw[X]
```

Comment:
Can you keep these on one line - betaflight style is not to wrap lines unless they are very long.